dbt_copilot_helper/commands/bootstrap.py

```diff
@@ -101,9 +125,13 @@ def load_and_validate_config(path):
 
     # validate the file
     schema = Schema(config_schema)
-    config = schema.validate(conf)
```

Comment:
this line was reordering the config (all nested dicts were moved to the bottom), which I don't think was desired behaviour.